packages/three-vrm-core/src/expressions/VRMExpressionLoaderPlugin.ts

```diff
@@ -323,7 +325,57 @@ export class VRMExpressionLoaderPlugin implements GLTFLoaderPlugin {
 
         const materialValues = schemaGroup.materialValues;
         if (materialValues && materialValues.length !== 0) {
-          console.warn('Material binds of VRM 0.0 are not supported. Setup the model in VRM 1.0 and try again');
+          materialValues.forEach((materialValue) => {
+            if (
+              materialValue.materialName === undefined ||
+              materialValue.propertyName === undefined ||
+              materialValue.targetValue === undefined
+            ) {
+              return;
+            }
+
+            const materials: THREE.Material[] = [];
+            gltf.scene.traverse((object) => {
+              if ((object as any).material) {
+                const material: THREE.Material[] | THREE.Material = (object as any).material;
+                if (Array.isArray(material)) {
+                  materials.push(
+                    ...material.filter(
+                      (mtl) => mtl.name === materialValue.materialName! && materials.indexOf(mtl) === -1,
```

Comment:
ここのtype assertionいらなさそう
```suggestion
                      (mtl) => mtl.name === materialValue.materialName && materials.indexOf(mtl) === -1,
```

Reply:
あー違うわ、filter関数内のラムダだからビックリ必要なんだこれ